contracts/PensionFundRelease.sol

```diff
@@ -7,10 +7,16 @@ contract PensionFundRelease {
     address[] public validators;
     address public worker;
     uint8 public firstPaymentPercent;
-    uint public firstPaymentTime;
+    uint8 public reccurentPaymentPercent;    
+    uint public paymentTime;
+    uint public lastPaymentTime;
     uint public reccurentPaymentInterval;
-    uint8 public reccurentPaymentPercent;
+    uint public totalPayment;
```

Comment:
Yeah sorry, those were from when I had it implemented before I had a clarification on how it should work, I'll pull them out.